docs/tutorials/remote_client.md

```diff
@@ -1,88 +1,111 @@
+This will add a remote connection to Podman and if it is the first connection added, it will mark the connection as the default.  You can observe your connections with ‘podman system connection list’
 
-### Remote node setup
-
-#### Initiate an ssh session to the Podman node
-To use the remote client, an ssh connection to the Podman server must be established.
+```
+$ podman-remote system connection list
+Name	  Identity 	       URI
+myuser*	  id_rsa	       ssh://myuser@192.168.122.1/run/user/1000/podman/podman.sock
+```
 
-Using the varlink bridge, an ssh tunnel must be initiated to connect to the server. Podman must then be informed of the location of the sshd server on the targeted server
+Now we can test the connection with ‘podman info’.
 
 ```
-$ export PODMAN_VARLINK_BRIDGE=$'ssh -T -p22 root@remotehost -- "varlink -A \'podman varlink \$VARLINK_ADDRESS\' bridge"'
-$ bin/podman-remote images
-REPOSITORY                     TAG      IMAGE ID       CREATED         SIZE
-docker.io/library/ubuntu       latest   47b19964fb50   2 weeks ago     90.7 MB
-docker.io/library/alpine       latest   caf27325b298   3 weeks ago     5.8 MB
-quay.io/cevich/gcloud_centos   latest   641dad61989a   5 weeks ago     489 MB
-k8s.gcr.io/pause               3.1      da86e6ba6ca1   14 months ago   747 kB
+$ podman-remote info
+host:
+  arch: amd64
+  buildahVersion: 1.16.0-dev
+  cgroupVersion: v2
+  conmon:
+	package: conmon-2.0.19-1.fc32.x86_64
 ```
 
-The PODMAN_VARLINK_BRIDGE variable may be added to your log in settings. It does not change per connection.
+Podman-remote has also introduced a “--connection” flag where you can use other connections you have defined.  If no connection is provided, the default connection will be used.
 
-If coming from a Windows machine, the PODMAN_VARLINK_BRIDGE is formatted as:
 ```
-set PODMAN_VARLINK_BRIDGE=C:\Windows\System32\OpenSSH\ssh.exe -T -p22 root@remotehost -- varlink -A "podman varlink $VARLINK_ADDRESS" bridge
+$ podman-remote system connection --help
 ```
 
-The arguments before the `--` are presented to ssh while the arguments after are for the varlink cli. The varlink arguments should be copied verbatim.
- - `-p` is the port on the remote host for the ssh tunnel.  `22` is the default.
- - `root` is the currently supported user, while `remotehost` is the name or IP address of the host providing the Podman service.
- - `-i` may be added to select an identity file.
+## Wrap up
+
+You can use the podman remote clients to manage your containers running on a Linux server.  The communication between client and server relies heavily on SSH connections and the use of SSH keys are encouraged.  Once you have Podman installed on your remote client, you should set up a connection using `podman-remote system connection add` which will then be used by subsequent Podman commands.
+
+## History
+Adapted from the [Mac and Windows tutorial](https://github.com/containers/podman/blob/master/docs/tutorials/mac_win_client.md)
```

Comment:
This might be a good blog on it's own.  You might run it by Jason and Tyler at enable sysadmin.